jekyll/_cci2/ios-migrating-from-1-2.md

```diff
@@ -100,7 +100,7 @@ jobs:
           name: Build IPA
           command: fastlane gym
 
-      # Store the IPA file in the {% comment %} TODO: Job {% endcomment %}build artifacts
+      # Store the IPA file in the job's artifacts
```

Comment:
follow-up on previous comment: because if `build` is the word we're using to describe an individual run of a job, then the artifacts are unique to that build, not to that job

Reply:
Good catch  — and related to your above question about the heading "Saving the Artifacts for a Build"

Reply:
I am not following -- the change seems correct as-is.

Reply:
Is an individual run of a job called a build?